website/i18n/en.json

```diff
@@ -214,6 +217,9 @@
         "title": "Amazon Kinesis ingestion",
         "sidebar_label": "Amazon Kinesis"
       },
+      "development/extensions-core/druid-kubernetes": {
```

Comment:
What? Why?

Reply:
O strange, I this was apparently auto-generated by running spellcheck locally, I just confirmed this by running ```npm run lint && npm run spellcheck``` on master and I get the same diff on this file

Reply:
Weird, but, okay. Next time please do this stuff in a separate commit so we can keep them clean.

Reply:
:+1:, I didn't actually realize it was there until you pointed it out, i just ran the spell check to make sure i fixed the errors in the docs and staged and pushed up all changes without really looking at them 😅 

Any idea what this file is for? Something with localization i would guess, but not sure if it is important, like should it be another commit to get pulled in to 0.21.0? Though I guess its possible that since building the website-src pulls in druid and does this build, that it might pick up the changes that happen to this file anyway as a side-effect.